person.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'corrector'
+
 class Person
```

Comment:
- [ ] `Person` class should inherits from `Nameable` class. It should also have a method `correct_name`:
```ruby
class Person < Nameable
  .
  .

    def correct_name
       name
    end
  .
  .
end
```


Kindly ensure to improve upon this aspect.